js/ElementRendererProvider.js

```diff
@@ -528,6 +550,98 @@ export default class ElementRendererProvider {
       return divCarouselWrapper;
     });
 
+    this.set(TYPES.CAROUSELSELECT, (config): HTMLElement => {
+      const defaultPadding = 0;
+      const padding = config.padding || defaultPadding;
+
+      const carouselWrapper = document.createElement('div');
+      const carousel = document.createElement('div');
+
+      if (config.accessibility && config.accessibility.web) {
+        Utils.appendAttributesFromObject(carouselWrapper, config.accessibility.web);
+      }
+
+      function findCardParent(element: any): HTMLDivElement | typeof undefined {
+        if (!element || element.tagName === 'BUTTON') {
+          return undefined;
+        }
+
+        const index = element.getAttribute('data-carousel-index');
+        if (index !== null) {
+          return element;
+        }
+
+        return findCardParent(element.parentNode);
+      }
+
+      function toggleCardSelect(element: HTMLDivElement, selected: boolean) {
+        if (selected) {
+          element.setAttribute('data-selected', 'true');
+          element.classList.add('lp-json-pollock-layout-selected');
```

Comment:
In addition to getting the selected border color from this default class, the selected border-color also needs the option to be able to be set via the json-schema too. Maybe something like `border-color-selected`(?)

Reply:
Done, thank you! I have updated the config, added the `styles` node into the schema, and extended `styles.json` schema to include `border-color-selected` property.

Now it uses this when available. If not, it will fallback to `class` I created